src/Serilog.Sinks.Console/LoggingExtentsions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Serilog.Configuration;
+using Serilog.Core;
+using Serilog.Events;
+using Serilog.Formatting.Display; 
+
+namespace Serilog.Sinks.SystemConsole
+{
+    public static class LoggingExtentsions
```

Comment:
I think we use `*LoggerConfigurationExtensions` in most other sinks - so `ConsoleLoggerConfigurationExtensions` here?